mobius/src/orthogonal_arcs.rs

```diff
@@ -0,0 +1,176 @@
+use std::f64::consts::{PI, TAU};
+
+use crate::{
+    geometry::{ArcAngles, Circle, CircularArc, GeneralizedCircle, Line, LineSegment},
+    nearly::is_nearly,
+    Complex,
+};
+
+#[derive(Debug)]
+pub enum OrthogonalArcError {
+    DuplicatePoint(Complex),
+    PointAtCenter(Complex),
```

Comment:
Looking at this again, I don't like that it's possible to pass in a point not on the original circle. Specifying an ArcAngles would be cleaner I think